### DISCUSSION TITLE
<titlDisco Diffusion Srops running and will not loade>

### DISCUSSION THREAD

**Discussion by Ralph717** [community/community#42578] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have tried all day to start disco Diffusion to render a set of very clear instructions which I have used in the past.  today it does not work and stops running after a few minutes when it gets to load 32S

**Comment by jfullstackdev** [comment, marked as answer]
if this is the same project 

https://github.com/alembics/disco-diffusion

then you can raise an issue and start asking there

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Ralph717! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
